website/pages/en/index.js

```diff
@@ -408,37 +295,37 @@ const UsersSection = ({ language }) => {
         >
           <div style={{ display: "flex", marginTop: "22px" }}>
             <a
-              href="https://npmjs.com/package/prettier"
+              href="https://github.com/prettier/prettier"
               className="growOnHover"
             >
-              <img src="/images/npm_grey.svg" style={{ height: "100px" }} />
+              <img src="/images/github_grey.svg" style={{ height: "100px" }} />
             </a>
             <div style={{ marginLeft: ".7em", width: "260px" }}>
               <p>
-                More than <strong data-placeholder="dependent-npm">5000</strong>{" "}
-                tools and integrations on npm
+                More than{" "}
+                <strong data-placeholder="dependent-github">1.7M</strong>{" "}
```

Comment:
```suggestion
                <strong data-placeholder="dependent-github">1.7 million</strong>{" "}
```

To emphasize the popularity of Prettier